lib/components/table.vue

```diff
@@ -344,10 +342,14 @@
                     return [];
                 }
 
+                // Shallow copy of items, so we don't mutate the original array order/size
                 items = items.slice();
 
                 // Apply local filter
                 if (filter && !this.providerFiltering) {
+                    // Number of Items before filtering
+                    let numOriginalItems = items.length;
+
```

Comment:
Ah, a leftover when I was overwriting the value in a previous commit.  I'll fix.